### PR TITLE
ODD tutorial

### PR DESCRIPTION
**Pull request by kepper**
This is a new tutorial on ODD, as used for MEI. Originally, I hope to have a second tutorial on customizations ready as well, but that needs a little more time, and I think this one should go first, and I'll sneak in the other one asap… 

I also apologize for not leaving sufficient time for reviewers. I will merge this PR in a couple of hours, as I need it for a workshop. Of course, changes are more than welcome, even after it has been merged…

One question to think of: I'm not perfectly sure that a tutorial is the right format for this, but the Guidelines don't seem like a better fit. It might also benefit from some additional diagrams…

### CODE REVIEW

**Review by musicEnfanthen — CHANGES_REQUESTED** [review]
Thanks for this impressive work. 

It surely needs some polishing in the future (wording, structure), but should be fine for your purpose now. Added some minor observations. Just went through the files and did not run the tutorial locally.

Good luck for your workshop :)

**Review by lpugin — APPROVED** [review]
Approving @musicEnfanthen review

**Comment by musicEnfanthen** [comment]
Just one more thing: For security reasons (see https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/), I'd recommend to replace all

`target="_blank"`

with 

`target="_blank" rel="noopener, noreferrer"`

as it is already in other tutorial steps

File: _tutorials/180_understanding-odd/understanding-odd-09.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                        &lt;rng:text/&gt;
+                        &lt;rng:ref name="model.textPhraseLike.limited" /&gt;
+                    &lt;/rng:choice&gt;
+                &lt;/rng:zeroOrMore&gt;
+            &lt;/rng:choice&gt;
+        &lt;/content&gt;
+    &lt;/macroSpec&gt;</pre>
+    <p>
+        Macros again use the <a href="https://tei-c.org/release/doc/tei-p5-doc/en/html/ref-macroSpec.html" target="_blank"><em>&lt;macroSpec&gt;</em></a>
+        element, but this time with a <em>@type</em> with value "<em>pe</em>". The names, which are again stored in <em>@ident</em>,
```
[the remark this answers is not in the document]
Comment (kepper) [reply _tutorials/180_understanding-odd/understanding-odd-09.html:30]: I have no clue, and in current ODD, it's not even allowed anymore ;-) Therefore, I intentionally left it out here. Maybe @pe-ro or @peterstadler can help out?

**Comment by kepper** [comment]
Thanks @musicEnfanthen for your wonderful review. Will try to sneak all that in and get it online asap. And thanks again for the noreferrer reminder – I always forget about this…

**Review by musicEnfanthen — APPROVED** [review]

**Pull request merged**